api/controllers/projectController.js

```diff
@@ -0,0 +1,61 @@
+import ProjectService from "../services/projectService.js";
+
+class ProjectController {
+  static async getAllProjects(req, res) {
+    try {
+      const projects = await ProjectService.getAllProjects();
+      res.status(200).json(projects);
+    } catch (error) {
+      res.status(500).json({ error: error.message });
```

Comment:
same comment for all other error messages. Unless you want to specific error message exposed to the client

Reply:
i added the specific errors in the projectService.js and from there i catch the message in the controller, was that what you meant?

Reply:
I see, then it's fine